### ISSUE TITLE
.add(error.scalarMultiply(5 / 16)).clip(0, 255).toRGB())))

### ISSUE THREAD

**Issue by yangfeifei1994** [Regarrzo/Java-Floyd-Steinberg-Dithering#1]
Shouldn't we cast 5/16 to (float) here? It seems like this parameter is getting lost. Without (float), the value here ends up being 0.

**Comment by Regarrzo** [comment]
Yes, that is correct. Thank you for pointing out the mistake, I will fix it!
